### PR TITLE
Udp web interface

### PR DESCRIPTION
**Pull request by Jeberlen**
# Description 
We added some more controls to the interface, including a role swapping functionality.

## Added functionality
- Role switch
- Terminal controller in web (WIP)

### On your computer
docker build --no-cache -f  Dockerfile.amd64 -t webinterface .
docker run --rm -ti --net=host -p 5001:5001 webinterface:latest --cid=122
### On/to the car
docker build --no-cache -f  Dockerfile.armhf -t webinterface
docker run --rm -ti --net=host -p 5001:5001 webinterface:latest --cid=122

## Verification/validation
Manual testing using console logs, to make sure the message is constructed properly and the web interface looks and feels nice.

### CODE REVIEW

File: src/web-interface/src/libcluon-viewer-0.01.js [hunk 1]
```diff
@@ -335,6 +454,7 @@ function addFieldCharts(sourceKey, data) {
         }
       };
       
+      console.log("source %s %s", sourceKey, i)
```
Comment (IsabelleTornqvist) [inline src/web-interface/src/libcluon-viewer-0.01.js]: console.log() - for debugging or useful?

File: src/web-interface/src/messages.odvd [hunk 2]
```diff
@@ -1,17 +1,47 @@
...
+
+/********************************************************/
+/** Internal Messages ***********************************/
+/********************************************************/
+
+message Role [id = 2203] {
+  uint8 current [id = 1];
+}
+
+message opendlv.proxy.AccelerationReading [id = 1030] {
```
Comment (IsabelleTornqvist) [inline src/web-interface/src/messages.odvd]: My bad, shouldn't be included (the .AccelerationReading)

Comment (Jeberlen) [reply src/web-interface/src/messages.odvd]: Are we not using this for later?

File: src/web-interface/src/libcluon-viewer-0.01.js (hunk 1 above)
Comment (Jeberlen) [reply src/web-interface/src/libcluon-viewer-0.01.js]: Can still be useful for future debugging, after merge

**Comment by guslauer** [comment]
- We can have leader as role on launch!
- When we are in following mode, we can still send commands by pressing A W D. It is not a real problem I guess, since we catch that in our maneuvering script where we filter it out. But maybe disable the keys on following mode.
- We probably need to add the slider back for steering angle............ :) Since we must demo a 90 degfree turn, and turning at 0.38 will take some time!
- Finally, maybe disable certain V2V functions base on the role we currently are in. For instance, I guess we should not be able to follow request when in following mode.

File: src/web-interface/src/messages.odvd (hunk 2 above)
Comment (IsabelleTornqvist) [reply src/web-interface/src/messages.odvd]: Not that I'm aware of, but I could be wrong ofc

**Pull request merged**